### PR TITLE
Add Haploview v4.2

### PR DESCRIPTION
**Pull request by simojoe**
Add Haploview v4.2

Haploview is presently distributed as a single jar file. 
I reused the wrapper code from the peptide-shaker recipe presented in the contribution guidelines.

### CODE REVIEW

**Comment by simojoe** [comment]
@BiocondaBot please add label

File: recipes/haploview/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{% set name = "haploview" %}
+{% set version = "4.2" %}
+
+about:
```
Comment (bgruening) [inline recipes/haploview/meta.yaml:4]: can you please move this down, so that it follows the ordering of other recipes? Thanks!

**Comment by simojoe** [comment]
Thanks!
I ordered by taking example on the peptide-shaker recipe.
I reordered using the kallisto recipe as a base design.

**Review by bgruening — APPROVED** [review]
Cool, thanks!

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**